docs/pages/reference/mesheryctl/mesheryctl-pattern-list.md

```diff
@@ -38,8 +38,9 @@ mesheryctl pattern list
 
 <pre class='codeblock-pre'>
 <div class='codeblock'>
-  -h, --help      help for list
-  -v, --verbose   Display full length user and pattern file identifiers
+  -h, --help       help for list
```

Comment:
@dusdjhyeon this auto-generated I don't think we need to pass this in a PR.
Here it is done with this [GH action](https://github.com/meshery/meshery/actions/runs/9744810579).